src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -915,4 +915,42 @@ DocumentUpload findDocumentUploadInEvent(final ProcurementEvent event, final Str
             String.format(ERR_MSG_FMT_DOCUMENT_NOT_FOUND, documentId)));
   }
 
+  /**
+   * Export buyer attachments
+   * 
+   * @param procId
+   * @param eventId
+   * @param principal
+   * @return list of attachments
+   */
+  public List<DocumentAttachment> exportDocuments(final Integer procId, final String eventId,
+      final String principal) {
+    log.debug("Export all Documents from Event {}", eventId);
+    var event = validationService.validateProjectAndEventIds(procId, eventId);
+    var exportRfxResponse = jaggaerService.getRfx(event.getExternalEventId());
+    var status = jaggaerAPIConfig.getRfxStatusToTenderStatus()
+        .get(exportRfxResponse.getRfxSetting().getStatusCode());
+    var attachments = new ArrayList<DocumentAttachment>();
+
+    if (TenderStatus.ACTIVE != status) {
+      // Get documents from S3
+      event.getDocumentUploads().stream().forEach(doc -> {
+        var documentKey = DocumentKey.fromString(doc.getDocumentId());
+        var attachment = DocumentAttachment.builder()
+            .data(documentUploadService.retrieveDocument(doc, principal))
+            .fileName(documentKey.getFileName())
+            .contentType(MediaType.parseMediaType(doc.getMimetype())).build();
+        attachments.add(attachment);
+      });
+    } else {
+      // Get documents from Jaggaer
+      exportRfxResponse.getBuyerAttachmentsList().getAttachment().stream()
```

Comment:
I've published an event with 3 uploaded files, and 4 generated `FC` documents - I can see them on the ITT (`itt_14630`) but when I invoke the export endpoint I am getting a 0 length response back - can you reproduce that?